domain-services/ifix-master-data-service/src/main/resources/application.properties

```diff
@@ -21,10 +21,9 @@ spring.kafka.producer.value-serializer=org.springframework.kafka.support.seriali
 
 #mongoDB
 ##TO DO - Update the host address as per environment
-#spring.data.mongodb.host=localhost
-#spring.data.mongodb.port=27017
+spring.data.mongodb.host=localhost
+spring.data.mongodb.port=27017
 spring.data.mongodb.database=ifix_dev_db
-spring.data.mongodb.url=mongodb://localhost:27017/?retryWrites=false
```

Comment:
We need to configure url env var. The combination of host and port won't work on server.

Reply:
Will make the changes.